### PR TITLE
esp32 rx - allow non uart0 pins for serial setup

### PR DESCRIPTION
**Pull request by JyeSmith**
When the rx and tx pins are changed via the hardware UI, they now actually change.

### CODE REVIEW

File: src/src/rx_main.cpp [hunk 1]
```diff
@@ -1246,7 +1246,7 @@ static void setupSerial()
     Serial.begin(serialBaud, config, mode, -1, invert);
 #elif defined(PLATFORM_ESP32)
     uint32_t config = sbusSerialOutput ? SERIAL_8E2 : SERIAL_8N1;
-    Serial.begin(serialBaud, config, -1, -1, invert);
+    Serial.begin(serialBaud, config, GPIO_PIN_DEBUG_RX, GPIO_PIN_DEBUG_TX, invert);
```
Comment (pkendall64) [inline src/src/rx_main.cpp:1249]: Shouldn't these be `GPIO_PIN_RCSIGNAL_RX` and `GPIO_PIN_RCSIGNAL_TX`? In which case the other change is not required.

Comment (JyeSmith) [reply src/src/rx_main.cpp]: FFS... fixed 😄

**Review by pkendall64 — APPROVED** [review]
You do realise that I will probably do another major rework on this after the 3.3 release as well 😆

**Review by wvarty — APPROVED** [review]
Tested on iFlight 900mhz TX, flashed as an RX, with pins 13 and 15. Working both as a CRSF output as well as AirPort.

**Pull request merged**